apps/apps.c

```diff
@@ -2744,3 +2526,14 @@ void make_uppercase(char *string)
     for (i = 0; string[i] != '\0'; i++)
         string[i] = toupper((unsigned char)string[i]);
 }
+
+int opt_vprintf_stderr(const char *fmt, ...)
```

Comment:
Intuitively, this is a bad name, as the 'v' prefix is normally used with functions that take a `va_list` instead of stdarg.  Just remove the 'v' and you'll be fine.